plugins/inputs/statsd/README.md

```diff
@@ -43,6 +43,7 @@
   ## Parses tags in the datadog statsd format
   ## http://docs.datadoghq.com/guides/dogstatsd/
   parse_data_dog_tags = false
+  parse_data_dog_events = false
```

Comment:
also... do you think it is possible to document the events translation in the readme?